schema/queue/MQcommon/Publisher.go

```diff
@@ -68,7 +68,7 @@ func (pb *Publisher) Publish(body []byte, routingKey string, exchange string, th
 
 		return err
 	}
-	pb.Logger.Info().Msg(" [x] Sent ")
+	pb.Logger.Info().Msg("MessageGroupBatch Published")
```

Comment:
Please, use a lower logging level (I think the `TRACE` level will be more suitable)